base/repl/LineEdit.jl

```diff
@@ -599,6 +600,32 @@ function edit_transpose(buf::IOBuffer)
     return true
 end
 
+edit_transpose_words(s) = edit_transpose_words(buffer(s)) && refresh_line(s)
+
+function edit_transpose_words(buf::IOBuffer, mode=:emacs)
+    mode in [:readline, :emacs] ||
```

Comment:
would it be possible to describe what the difference in behavior is in words somewhere? this doesn't have a docstring, but a comment would do if anyone is going to be customizing these